scss/_maps.scss

```diff
@@ -112,13 +112,15 @@ $utilities-colors: $theme-colors-rgb !default;
 // scss-docs-end utilities-colors
 
 // scss-docs-start utilities-text-colors
-// Boosted mod: content of $utilities-text
-$utilities-text: (
-  "primary": to-rgb($accessible-orange),
-  "light": to-rgb($gray-500),
-  "black": to-rgb($black),
-  "white": to-rgb($white),
-  "body": to-rgb($body-color),
+$utilities-text: map-merge(
+  $utilities-colors,
+  (
+    "primary": to-rgb($accessible-orange),
+    "light": to-rgb($gray-500),
```

Comment:
I let you double-check, but I'd say that `"light"` doesn't need to be redefined. With some logs, I got the following, and it seems to already have the right value in `$utilities-colors`:

```
scss/_maps.scss:112 Debug: "utilities-colors", ("primary": (255, 121, 0), "secondary": (0, 0, 0), "success": (50, 200, 50), "info": (82, 126, 219), "warning": (255, 204, 0), "danger": (205, 60, 20), "light": (204, 204, 204), "dark": (0, 0, 0))

scss/_maps.scss:126 Debug: "utilities-text", ("primary": (241, 110, 0), "secondary": (0, 0, 0), "success": (50, 200, 50), "info": (82, 126, 219), "warning": (255, 204, 0), "danger": (205, 60, 20), "light": (204, 204, 204), "dark": (0, 0, 0), "black": (0, 0, 0), "white": (255, 255, 255), "body": (0, 0, 0))
```